engines/sci/resource_audio.cpp

```diff
@@ -469,6 +469,12 @@ int ResourceManager::readAudioMapSCI11(IntMapResourceSource *map) {
 				// FIXME: The sync36 resource seems to be two bytes too big in KQ6CD
 				// (bytes taken from the RAVE resource right after it)
 				if (syncSize > 0) {
+					// LB2CD is missing the sync resource for message 1885 1 6 30 2 but it's a duplicate
+					//  of 1885 1 6 16 2 which does have a sync resource so use that for both. bug #9956
+					if (g_sci->getGameId() == GID_LAURABOW2 && map->_mapNumber == 1885 && n == 0x01061002) {
```

Comment:
Please add "g_sci->isCD()" to make sure it's the CD version.

Overall I haven't looked into this. Is a script patch impossible? I would prefer a script patch in these cases, if possible.
I think at one point we should add some mechanic to replace resources/fix resource data in a way. That way we could also fix the other broken lip sync data.

Reply:
Only the CD version has sync resources, so adding a CD check here is unnecessary